core/workspace_svg.js

```diff
@@ -509,6 +522,9 @@ Blockly.WorkspaceSvg.prototype.translate = function(x, y) {
       'scale(' + this.scale + ')';
   this.svgBlockCanvas_.setAttribute('transform', translation);
   this.svgBubbleCanvas_.setAttribute('transform', translation);
+  if (this.blockDragSurface_) {
+    this.blockDragSurface_.translateAndScaleGroup(x, y, this.scale);
```

Comment:
Does this need to be updated every time the workspace translates?

I guess the advantage is that we don't have this slowing down block drag starts, but doesn't it make every workspace movement slower?